Library/Homebrew/dev-cmd/edit.rb

```diff
@@ -50,42 +50,47 @@ def edit
         [HOMEBREW_REPOSITORY]
       end
     else
+      edit_api_message_displayed = false
       args.named.to_paths.select do |path|
-        next path if path.exist?
+        fc_api_path = path.fnmatch?("**/homebrew-core/Formula/*.rb") || path.fnmatch?("**/homebrew-cask/Casks/*.rb")
```

Comment:
What does `fc` mean? As I have no idea: can you use a better/longer name? Thanks ❤️